### ISSUE TITLE
TypeError: 'dict_values' object does not support indexing

### ISSUE THREAD

**Issue by kulturlupenguen** [koalalorenzo/python-digitalocean#79]
I use Python 3.4 in Windows and I got this error while I was trying to list all of user images. Traceback is here:

```python-traceback
Traceback (most recent call last):
  File "C:\Users\YusufTuğrul\Desktop\digitalocean.py\digitaloceancli.py", line 216, in <module>
    DigitalOcean().cmdloop()
  File "C:\Python34\lib\cmd.py", line 138, in cmdloop
    stop = self.onecmd(line)
  File "C:\Python34\lib\cmd.py", line 217, in onecmd
    return func(arg)
  File "C:\Users\YusufTuğrul\Desktop\digitalocean.py\digitaloceancli.py", line 204, in do_lsimages
    images = self.manager.get_all_images()
  File "C:\Python34\lib\site-packages\python_digitalocean-1.1-py3.4.egg\digitalocean\Manager.py", line 113, in get_all_images
    data = self.get_data("images/")
  File "C:\Python34\lib\site-packages\python_digitalocean-1.1-py3.4.egg\digitalocean\Manager.py", line 26, in get_data
    unpaged_data = self.__deal_with_pagination(args[0], data, params)
  File "C:\Python34\lib\site-packages\python_digitalocean-1.1-py3.4.egg\digitalocean\Manager.py", line 43, in __deal_with_pagination
    more_values = new_data.values()[0]
TypeError: 'dict_values' object does not support indexing
```

**Comment by koalalorenzo** [comment]
I can see that you are using python 3. I am going to fix this and release the new version ASAP.

**Comment by kulturlupenguen** [comment]
You're so fast. I wonder that what are the differences between Python 3 and Python 2 at dict objects? Can you explain these to me?

**Comment by koalalorenzo** [comment]
The code is using dict().values() method that is quite different between python 3 and python 2. In python 2 it returns a list, and in python 3 it doesn't. To fix this I had to "cast" the value returned into a List. So the if the value returned is not a list, it will be, in this way we are supporting both python 3 and python 2. I just released the version 1.2 and I think I fixed it. In case feel free to re-open the issue.

**Comment by kulturlupenguen** [comment]
Thanks for fix. It's working know.
